packages/web/components/cards/app-display-card.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+
+import { EventName } from "~/config";
+import { useAmplitudeAnalytics } from "~/hooks";
+
+import { Icon } from "../assets";
+import { IconLink } from "./icon-link";
+
+interface AppDisplayCardProps {
+  title?: string;
+  subtitle?: string;
+  imageUrl?: string;
+  twitterUrl?: string;
+  githubUrl?: string;
+  externalUrl?: string;
+  mediumUrl?: string;
+  index: number;
+}
```

Comment:
This is a nit: but as you may notice across the app, we typically define function components as closures with inline prop type defs, like:

`FunctionComponent<{ prop1: string }>`

FunctionComponent: more readable
Inlining: less code, helps draw attention to actual type defs that are meant to be reused and used as annotation for APIs.